binr/radare2/radare2.c

```diff
@@ -521,11 +521,10 @@ int main(int argc, char **argv, char **envp) {
 			break;
 		case '0':
 			zerosep = true;
-			//r_config_set (r.config, "scr.color", "false");
 			/* implicit -q */
 			r_config_set (r.config, "scr.interactive", "false");
 			r_config_set (r.config, "scr.prompt", "false");
-			r_config_set (r.config, "scr.color", "false");
+			r_config_set_i (r.config, "scr.color", COLORS_DISABLED);
```

Comment:
i would say COLOR_MODE_DISABLED, so this way follows a correct naming convention for bindings and such

Reply:
then why do you merge haha